platforms/ios/config.cmake

```diff
@@ -15,22 +16,25 @@ if (IOS_SDK_VERSION VERSION_LESS 11.0)
 endif()
 set(SQLITECPP_INTERNAL_SQLITE OFF CACHE BOOL "")
 
+# Headers must be absolute paths for the copy_if_different command on the
+# static library target, relative paths cause it to fail with an error.
 set(TANGRAM_FRAMEWORK_HEADERS
-  platforms/ios/framework/src/TangramMap.h
-  platforms/ios/framework/src/TGGeoPolyline.h
-  platforms/ios/framework/src/TGGeoPolygon.h
-  platforms/ios/framework/src/TGGeoPoint.h
-  platforms/ios/framework/src/TGMarker.h
-  platforms/ios/framework/src/TGSceneUpdate.h
-  platforms/ios/framework/src/TGMapData.h
-  platforms/ios/framework/src/TGTypes.h
-  platforms/ios/framework/src/TGHttpHandler.h
-  platforms/ios/framework/src/TGLabelPickResult.h
-  platforms/ios/framework/src/TGMarkerPickResult.h
-  platforms/ios/framework/src/TGMapViewController.h
+  ${PROJECT_SOURCE_DIR}/platforms/ios/framework/src/TangramMap.h
```

Comment:
In previous changes I've preferred specifying source files by relative paths because 1) it's more concise and 2) if we decide to move a directory within the project the source files will remain at the same relative location to the CMakeLists.txt. However some CMake functions seem to only work correctly with absolute paths. Probably not a big enough deal to enforce uniformity, but good to be aware of.

Reply:
I am fine having it like this, until I can find my source (and if its a legit source). Cmake documentation does not mention anything specific on this. So its fine as it is.